libraries/AP_Math/tests/test_math.cpp

```diff
@@ -3,13 +3,12 @@
 // you're doing when directly comparing floats:
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wfloat-equal"
-
+#define ALLOW_DOUBLE_MATH_FUNCTIONS
```

Comment:
I'd really rather not!

Feel free to create another test_math_double.cpp if you want.